ApollosProject/apollos-apps
# Remove headers from summary

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR removes any headers from the text that could display in the post summary on a card.

**Before:**
![Screenshot 2020-03-03 15 54 18](https://user-images.githubusercontent.com/3229463/75820012-97537600-5d69-11ea-896f-0242f134b0bf.png)

**After:**
![Screenshot 2020-03-03 15 55 58](https://user-images.githubusercontent.com/3229463/75820013-97ec0c80-5d69-11ea-8540-edc109647095.png)

### What design trade-offs/decisions were made?
N/A

### How do I test this PR?
Look at the "This March at NewSpring" content item and make sure that the "Connect Classes" header does not show in the summary.

### What automated tests did you write?
None.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
